src/features/gtlPlaylist/components/GtlPlaylist.tsx

```diff
@@ -29,6 +29,8 @@ export function GtlPlaylist({ venue, playlist }: GtlPlaylistProps) {
     ({ item, width, height }) => {
       const hit = transformOfferHits(item)
       const timestampsInMillis = item.offer.dates?.map((timestampInSec) => timestampInSec * 1000)
+      const entryId = playlist.entryId
+      const itemIndex = playlist.offers.hits.findIndex((i) => i.objectID === hit.objectID)
```

Comment:
```suggestion
    ({ index, item, width, height }) => {
      const hit = transformOfferHits(item)
      const timestampsInMillis = item.offer.dates?.map((timestampInSec) => timestampInSec * 1000)
      const entryId = playlist.entryId
```

Simplement :) Tu peux récupérer `index` directement dans les props du callback, pas besoin de mapper dans toutes les `playlist.offers.hits`

Reply:
Ah oui bien vu